builds/jenkins-build-sls-app/Jenkinsfile

```diff
@@ -0,0 +1,1062 @@
+          def randomString = utilModule.generateRequestId();
+          def credsId = "jazz-${randomString}";
+          sh "aws configure set profile.${credsId}.region ${configLoader.AWS.REGION}"
+          sh "aws configure set profile.${credsId}.aws_access_key_id $AWS_ACCESS_KEY_ID"
+          sh "aws configure set profile.${credsId}.aws_secret_access_key $AWS_SECRET_ACCESS_KEY"
+
+          // Generate serverless yml file with domain added in function name
+          echo "Generate deployment env with domain"
+
+          if (isScheduleEnabled) {
+            def eventsArns = getEventsArn(config, environment_logical_id)
+            for (def i = 0; i < eventsArns.size(); i++) {
+              def arn = eventsArns[i]
+              events.sendCompletedEvent('CREATE_ASSET', null, utilModule.generateAssetMap("aws", arn, "cloudwatch_event", config), environment_logical_id);
+            }
+          }
+
+          def envBucketKey = "${env_key}${configLoader.JAZZ.S3_BUCKET_NAME_SUFFIX}"
+          echoServerlessFile()
+          def deployOutput = servelessDeploy(environment_logical_id, envBucketKey)
+
+// Shoving deploy output lines into array
+          def deployLines = deployOutput.split('\n')
+          def resourceArns = []
+          def afterStackOutputs = false
+// Extracting all the lines that follow 'Stack Outputs'. These lines are supposed to contain arns of resources created
+          for(def line: deployLines) {
+            if(afterStackOutputs) resourceArns.add(line)
+            if(line.contains('Stack Outputs')) afterStackOutputs = true // Here we are only interested in lines that follow 'Stack Outputs' marker
+          }
+
+          for(def name2ArnString: resourceArns) {
+            def name2ArnPair = name2ArnString.split(': ') // Here we rely on the space after the ': ' so it is split properly without affecting the arn itself that is full of colons but never has a space after
+            if(name2ArnPair.size() > 1) {
+              def arn = name2ArnPair[1].trim()
+              if(arn.startsWith('arn')) {
+                 def arnAsArray = arn.split(':') // Here we splitting the arn itself in hope to obtain type of the resource. Should be the third element: arn:aws:dynamodb:us-east-1:102707241671:table/jazzUsersTable53
+                 if(arnAsArray.size() > 2) { // Making sure that the third element exists
+                   def artifactType = arnAsArray[2]
+                   if(artifactType == 'kinesis') artifactType = 'kinesis_stream' // 'kinesis_stream' needs to be instaead of simple 'kinesis' that did not get persisted :-( by itself
+                   events.sendCompletedEvent('CREATE_ASSET',
+                                             null,
+                                             utilModule.generateAssetMap("aws",
+                                                                         arn,
+                                                                         artifactType,
```

Comment:
Since we whitelist artifacts that can be created using sls deploy, let's make sure we add them to list of supported assets in assets table. 

https://github.com/tmobile/jazz/blob/master/core/jazz_assets/swagger/swagger.json#L1136
https://github.com/tmobile/jazz/blob/master/core/jazz_assets/config/global-config.json#L25

Reply:
Looking at global-config.json
```json
{
	"ASSET_TYPES": [
		"lambda",
		"apigateway",
		"cloudfront",
		"s3",
		"dynamodb",
		"cloudwatch_event",
		"swagger_url",
		"endpoint_url",
                 "dynamodb_stream",
                 "iam_role",
                 "sqs",
                 "kinesis_stream"
	]
}
```
Which looks like matching our trusty whitelist.yml in respect of resources, so I am not too sure whether any changes are needed here